packages/core/README.md

```diff
@@ -94,4 +94,37 @@ const result = await core.group('Do something async', async () => {
   const response = await doSomeHTTPRequest()
   return response
 })
+```
+
+#### Action state
+
+You can use this library to save state and get state for sharing information between a given wrapper action: 
+
+**action.yml**
+```yaml
+name: 'Wrapper action sample'
+inputs:
+  name:
+    default: 'GitHub'
+runs:
+  using: 'node12'
+  main: 'main.js'
+  post: 'cleanup.js'
+```
+
+In action's `main.js`:
+
+```js
+const core = require('@actions/core');
+
+core.saveState("pidToKill", 12345);
+```
+
+In action's `cleanup.js`:
+```js
+const core = require('@actions/core');
+
+var pid = core.getState("pidToKill");
```

Comment:
Will this example work?
I think it might be nice to have examples use best practices and be functional when copied.

`kill()` will require a `core.exec` call.

I'm thinking an example like:
```js
const core = require('@actions/core');
const exec = require('@actions/exec');

const pid = core.getState("pidToKill");

await exec.exec(`kill ${pid}`);
```

Reply:
https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal

Reply:
The example doesn't have access to a `child_process` in the post to kill though, it's specifying a `pid`

Reply:
i will change this to be `process.kill(pid)`